src/types/wdi5.types.ts

```diff
@@ -86,6 +86,7 @@ export type BTPAuthenticator = {
 
 export type BasicAuthAuthenticator = {
     provider: "BasicAuth"
+    basicAuthUrl?: Array<string>
```

Comment:
@vobu @monavari-lebrecht: basicAuthUrl is an Array. Should we rename it to `basicAuthUrls`? I am not sure. Sound a little bit awkward 😄

Reply:
yep, do it!